__tests__/app.test.js

```diff
@@ -224,3 +224,34 @@ describe('/api/reviews', () => {
       });
   });
 });
+
+describe('/api/comments/:comment_id', () => {
+  describe('DELETE', () => {
+    it('returns 204 and deletes the relevant comment from the table', () => {
```

Comment:
I'd suggest that this test doesn't test that it's been removed from the table. 
The choice branching out before you:
- delete that bit of the test description
- add a bit in your test to check that the comment is no longer there